Makefile

```diff
@@ -132,26 +131,24 @@ logs-join:
 logs-clean:
 	-rm -f robottelo_gw*.log
 
-uuid-check:  ## list duplicated uuids
+uuid-check:  ## list duplicated or empty uuids
 	$(info "Checking for empty or duplicated @id: in docstrings...")
-	scripts/check_duplicate_uuids.sh
+	@scripts/check_uuids.sh
 
-uuid-replace-duplicate:  ## list duplicated uuids
-	scripts/replace_dup_uuids.sh
-
-uuid-replace-empty:  ## list duplicated uuids
-	scripts/replace_empty_uuids.sh
+uuid-fix:
+	@scripts/fix_uuids.sh
 
 flake8:
 	$(info "Checking style and syntax errors with flake8 linter...")
-	@flake8 . --show-source
+	@flake8 $(shell git diff --name-only) --show-source
```

Comment:
I mean, that is a personal preference but if others don't worry about it then it is fine to leave instead of creating a separated target.

I've just pointed some of the usecase I personally have.